app/views/targeted_retention_incentive_payments/claims/check_eligibility_intro.html.erb

```diff
@@ -0,0 +1,51 @@
+<% content_for(
+  :page_title,
+  page_title(
+    "Check you're eligible for a targeted retention incentive payment",
+    journey: current_journey_routing_name,
+    show_error: @form.errors.any?
+  )
+) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Check you're eligible for a targeted retention incentive payment
```

Comment:
small thing but all quotation marks are supposed to be smart quotes. see https://design.tax.service.gov.uk/hmrc-content-style-guide/#:~:text=quotation%20marks,these%20in%20different%20operating%20systems.